.github/workflows/check.yml

```diff
@@ -20,34 +20,59 @@ jobs:
     if: github.event_name != 'push'
     name: Audit Dependencies 🕵️‍♂️
     uses: insightsengineering/r.pkg.template/.github/workflows/audit.yaml@main
-  r-cmd:
+  r-cmd-dev:
     if: github.event_name != 'push'
     name: R CMD Check (development) 🧬
     uses: insightsengineering/r.pkg.template/.github/workflows/build-check-install.yaml@main
     secrets:
       REPO_GITHUB_TOKEN: ${{ secrets.REPO_GITHUB_TOKEN }}
     with:
-      install-deps-from-package-repositories: "R-universe=https://pharmaverse.r-universe.dev/,CRAN=https://cloud.r-project.org/"
       package-subdirectory: package
       additional-env-vars: |
         NOT_CRAN=true
         QUARTO_PROFILE=development
         TLG_CATALOG_PKG_BUILD_RENDER=TRUE
       concurrency-group: development
+      deps-installation-method: setup-r-dependencies
+      lookup-refs: |
+        insightsengineering/random.cdisc.data
+        insightsengineering/formatters
+        insightsengineering/rlistings
+        insightsengineering/rtables
+        insightsengineering/tern
+        insightsengineering/tern.mmrm
+        insightsengineering/tern.rbmi
+        insightsengineering/teal.data
+        insightsengineering/teal.code
+        insightsengineering/teal.logger
+        insightsengineering/teal.reporter
+        insightsengineering/teal.slice
+        insightsengineering/teal.transform
+        insightsengineering/teal.widgets
+        insightsengineering/teal
+        insightsengineering/teal.modules.general
+        insightsengineering/teal.modules.clinical
+      skip-desc-dev: true
```

Comment:
@pawelru what is the purpose of this parameter?

Reply:
Ah, I see this is passed in here
https://github.com/insightsengineering/r.pkg.template/pull/260/files#diff-4db3492d6468586dd50add2ee731b86d3211cad713aa03e56c7d059d891a0cdaR147-R174

Reply:
Yes. It i passed to the `setup-r-dependencies` and utilised there. 
it has to be analysed in the context of the following parameter - `repository-list`. This points to the r-universe with universe of our devel packages. Hence we can skip the logic of identifying dev deps as this is essentially redundant.

Reply:
Amazing, thanks for the reply